common/flight-rpc/src/dns_resolver.rs

```diff
@@ -37,6 +37,7 @@ use tonic::transport::ClientTlsConfig;
 use trust_dns_resolver::TokioAsyncResolver;
 
 use crate::FlightClientTlsConfig;
+use crate::GrpcClientTlsConfig;
```

Comment:
Do we keep `FlightClientTlsConfig` for compatibility?

Reply:
The title of this PR is a bit vague, replace arrow-flight with gRPC is just for query->meta, query<-->query still use arrow-flight cause they need pass files.
We do this change because users want to do communication with metasrv directly, with gRPC we can expose API easily.